### PR TITLE
New Template: Cisco WLC show time (for time and NTP information)

### PR DESCRIPTION
**Pull request by mjbear**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

cisco_wlc_ssh_show_time.textfsm, cisco_wlc_ssh, show time

##### SUMMARY

Add support for parsing `show time` output from Cisco WLC so the time zone and NTP servers are accessible.

```

```

### CODE REVIEW

File: templates/cisco_wlc_ssh_show_time.textfsm [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Value TIME (.+)
```
Comment (FragmentedPacket) [inline templates/cisco_wlc_ssh_show_time.textfsm:1]: I would make `TIME`, `TZ_DELTA`, `TZ_NAME`, `NTP_VERSION`, `NTP_POLLING_INTERVAL` as `FILLDOWN`.

File: templates/cisco_wlc_ssh_show_time.textfsm [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Value STATUS (\S+\s?\S+)
+Value NTP_MSG_AUTH_STATUS (\S+\s?\S+)
+
+Start
+  ^\s*Time\s*\.+\s+${TIME}\s*$$
+  ^\s*Timezone [dD]elta\s*\.+\s+${TZ_DELTA}\s*$$
+  ^\s*Timezone [lL]ocation\s*\.+\s+\(${TIMEZONE}\)\s+${TZ_NAME}\s*$$
+  ^\s*NTP [sS]ervers\s*$$
+  ^\s*NTP [vV]ersion\s*\.+\s+${NTP_VERSION}\s*$$
+  ^\s*NTP [pP]olling [iI]nterval\s*\.+\s+${NTP_POLLING_INTERVAL}\s*$$ -> Record
```
Comment (FragmentedPacket) [inline templates/cisco_wlc_ssh_show_time.textfsm:19]: With the comment above, I'd remove this `Record` and just let it record at the NTP servers so it'll parse two entries with the same top level information.

**Review by FragmentedPacket — CHANGES_REQUESTED** [review]

File: templates/cisco_wlc_ssh_show_time.textfsm (hunk 1 above)
Comment (mjbear) [reply templates/cisco_wlc_ssh_show_time.textfsm]: Modified.

File: templates/cisco_wlc_ssh_show_time.textfsm (hunk 2 above)
Comment (mjbear) [reply templates/cisco_wlc_ssh_show_time.textfsm]: @FragmentedPacket 
I removed the Record so the entries/parsed data is more succinct. Thank you!
Oddly enough there's an extra line being picked up, but I can't seem to figure out what's adding it.

Comment (FragmentedPacket) [reply templates/cisco_wlc_ssh_show_time.textfsm]: Ah ok. Make the `index` required and I think that should fix the issue.

Comment (mjbear) [reply templates/cisco_wlc_ssh_show_time.textfsm]: > Ah ok. Make the `index` required and I think that should fix the issue.

That did it. Thank you!
Fixed up and changes have been pushed to my branch.

**Review by FragmentedPacket — APPROVED** [review]

**Pull request merged**